vignettes/queries_dataset.Rmd

```diff
@@ -0,0 +1,179 @@
+---
+title: "Queries dataset documentation"
+output:
+  rmarkdown::html_vignette:
+  toc: false
+vignette: >
+  %\VignetteIndexEntry{Queries dataset documentation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+  ```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+To support the safety analysis, it is quite common to define specific grouping of events. One of the most common ways is to group events or medications by a specific medical concept such as a Standard MedDRA Queries or WHO-Drug Standardized Drug Grouping. 
+
+
+To help with the derivation of these variables, the admiral function `derive_vars_query()` can be used.
+This function takes as input the dataset (`dataset`) where the grouping must occur (e.g `ADAE`) and
+a dataset containing the required information to perform the derivation of the grouping variables 
+(`dataset_queries`).
+
+Because the definition, collection and storage of the medical concepts is company specific, the dataset passed to the `dataset_queries` argument of the `derive_vars_query()` function should be created outside of admiral in a specific format so that it can be passed in the function.
+
+This vignette describes the expected structure and content of the dataset passed to the
+`dataset_queries` argument in the `derive_vars_query()` function. 
+
+# Structure of the Query Dataset.
+
+## Variables
+
+Variable | Scope |  Type | Example Value
+------- | ----- | ------ | ----- | ----- | -----
```

Comment:
You have more columns than headings here. Only the columns with a heading are displayed when knitting the document.

Reply:
updated